src/test/compile-fail/E0050.rs

```diff
@@ -9,13 +9,20 @@
 // except according to those terms.
 
 trait Foo {
-    fn foo(&self, x: u8) -> bool;
+    fn foo(&self, x: u8) -> bool; //~ NOTE expected 1 more parameter
+    fn bar(&self, x: u8, y: u8, z: u8); //~ NOTE expected 3 more parameters
+    fn less(&self); //~ NOTE expected 3 less parameters
 }
 
 struct Bar;
 
 impl Foo for Bar {
     fn foo(&self) -> bool { true } //~ ERROR E0050
+                                   //~| NOTE found 1 parameter
+    fn bar(&self) { } //~ ERROR E0050
+                      //~| NOTE found 1 parameter
+    fn less(&self, x: u8, y: u8, z: u8) { } //~ ERROR E0050
+                                            //~| NOTE found 4 parameters
```

Comment:
Rather than putting "found x parameters" here, I think the design in https://github.com/rust-lang/rust/issues/35211 would read better.  Specifically the primary errors showing specifically what is going wrong.